### ISSUE TITLE
Installation is stucking

### ISSUE THREAD

**Issue by ftrprf84** [lukeslater1911/Asus-Z690-Maximus-Hero-Hackintosh#2]
hey luke, just have a look. I tried all 3 versions of your EFI, pasted my all my serials at the generic part of your EFI and tried to start to install. And now my system is stopping at this point. What did I wrong? I followed your instructions exactly.

![screenshot_hackintosh](https://user-images.githubusercontent.com/126405660/222784894-a092611b-69b8-48b8-8d76-d9f2b094af27.jpg)

**Comment by lukeslater1911** [comment]
What BIOS version do you use? You need the 2204 version + you need to update the ME Firmware. That can only be done in windows. Maybe a windows PE boot stick can do this. I have a second ssd with a win11 install

**Comment by ftrprf84** [comment]
Yes, me too. The plan is to run on one M.2 Windows 11 and on the other m.2 ssd Mac. I will check the bios version. 

Does it matter which Mac OS install version i got on the stick? Because I got right now the Ventura beta install on it

**Comment by lukeslater1911** [comment]
make a new usb with the most actual Ventura version. I never tried the beta versions

**Comment by ftrprf84** [comment]
Alright, I will try a second stick with latest Ventura version. 

One thing: I'm using the "modified" EFI file. Can you recommend this version?

**Comment by ftrprf84** [comment]
Update: I got right now at the moment the 2103 bios version :-(

**Comment by lukeslater1911** [comment]
I just uploaded the 0.89 EFi. Best approach is to install Win11 first, then macOS

**Comment by ftrprf84** [comment]
Do I have to disable the wifi & bt controller in bios extra? or did you disabled it automaticly in your EFI?

**Comment by lukeslater1911** [comment]
no need to disable them

**Comment by ftrprf84** [comment]
Ok, good to know for Win11.

Still waiting since 10 minutes with that screen, I think it stopped here :-( What do you think?

![IMG_1534](https://user-images.githubusercontent.com/126405660/222799132-448c7f4c-4dcd-4a46-814d-4ef970b4e4e8.jpg)

**Comment by lukeslater1911** [comment]
Like I said, install win11 first. Windows doesn't like other bootloaders. After Win11 install update BIOS and ME. Then install macOS

**Comment by ftrprf84** [comment]
I got two separated SSD cards. One 1TB with Win11 on it and the other SSD for Mac OS. the mac ssd is totally blank, nothing on it. I don't have one ssd with two partions. I already updated bios with the Win11 SSD. The latest version 2204 is installed. After the update, I removed the win11 ssd from the m.2 port and installed the mac ssd

**Comment by ftrprf84** [comment]
Maybe it stucks because of the Ventura Beta install file. I try it with the latest, normal version of Ventura, no beta. I will keep you updated

**Comment by ftrprf84** [comment]
![98236E71-ACFA-409F-B5D1-19BDAF1050C8](https://user-images.githubusercontent.com/126405660/222925547-d125b8df-4f1e-47bd-a4f6-30209670aacd.jpeg)

Don‘t know what I‘m doin wrong. I followed your step exactly. Created a new bootable stick. Got the same parts as you do. Bios is on default. But the install process just stop with this line:

com.apple.xpc.launchd|2021-01-01 12:02:37 .755829 (system) <Warning›: failed lookup: name = com.apple.logd, flags = 0x9, requestor = locationd[66], error = 3: No such process com.apple.xp

I dont know what to do meanwhile…

**Comment by lukeslater1911** [comment]
Did you follow dortania's guide on how to generate serial and ROM info? If you don't have a valid ROM address install can't continue.
https://dortania.github.io/OpenCore-Install-Guide/config.plist/comet-lake.html#platforminfo
